aiida/engine/processes/ports.py

```diff
@@ -80,12 +80,12 @@ def __init__(self, *args, **kwargs) -> None:
         self._serializer: Callable[[Any], 'Data'] = serializer
 
     def serialize(self, value: Any) -> 'Data':
-        """Serialize the given value if it is not already a Data type and a serializer function is defined
+        """Serialize the given value if it is not ``None`` or already a Data type and a serializer function is defined.
```

Comment:
Maybe it is my language expression issue, but I think it is more clear to say:
"Serialize the given value if it is not ``None`` or not a Data type and a serializer function is not defined." ?

Reply:
Agreed, it is not super clear. I have improved it to
```
Serialize the given value, unless it is ``None``, already a Data type, or no serializer function is defined
```
I have also updated the docs, which I had forgot.